src/Sulu/Component/Content/Mapper/ContentMapper.php

```diff
@@ -770,7 +774,7 @@ private function copyOrMove($uuid, $destParentUuid, $userId, $webspaceKey, $loca
 
         $this->documentManager->find($document->getUuid(), $originalLocale);
 
-        return $this->documentToStructure($document);
+        return $this->documentToStructure($originalDocument);
```

Comment:
I am not sure about this, but wouldn't it make more sense to return the new document here?